Libs/RaptureCore/src/main/java/rapture/kernel/EventApiImpl.java

```diff
@@ -65,7 +65,8 @@ public EventApiImpl(Kernel raptureKernel) {
 
     @Override
     public List<RaptureFolderInfo> listEventsByUriPrefix(CallingContext context, String eventUriPrefix) {
-        return RaptureEventStorage.getChildren(eventUriPrefix);
+        int c = eventUriPrefix.indexOf(':');
+        return RaptureEventStorage.getChildren((c > 0) ? eventUriPrefix.substring(c+1) : eventUriPrefix);
```

Comment:
Just removing the scheme if it's present. If you passed in event://foo/bar it got changed to event/event://foo/bar